CMakeLists.txt

```diff
@@ -3,20 +3,28 @@ cmake_minimum_required(VERSION 3.28.2)
 # Get version
 file(READ ${CMAKE_CURRENT_SOURCE_DIR}/props.json PROPS)
 string(JSON VER GET ${PROPS} version)
-
+# find curl
+find_package(CURL REQUIRED)
 project(stc
   DESCRIPTION "Easily download collections and mods from steam."
   VERSION ${VER})
 
 set(CMAKE_CXX_STANDARD 26)
-
+# check if build type is set if it isnt set it to release
+if(NOT CMAKE_BUILD_TYPE)
+  set(DCKMAKE_BUILD_TYPE Release)
+endif()
 # Add the path to the source files for stc
 file(GLOB_RECURSE STC_SOURCE_FILES "src/*.cpp")
-
```

Comment:
I will prob set it to -02 instead cus -03 can cause issues sometimes

Reply:
I have been running gentoo on -O3 its really stable even while using gcc if you use a newer version